js/privatebin.js

```diff
@@ -2123,149 +2136,162 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         me.moveAttachmentTo = function($element, label)
         {
             // move elemement to new place
-            me.$attachmentLink.appendTo($element);
+            $attachmentLink.appendTo($element);
 
             // update text
-            I18n._(me.$attachmentLink, label, me.$attachmentLink.attr('download'));
-        }
-
+            I18n._($attachmentLink, label, $attachmentLink.attr('download'));
+        };
 
-        /*
-         * Read file data as dataURL using the FileReader API
-         * https://developer.mozilla.org/en-US/docs/Web/API/FileReader#readAsDataURL()
+        /**
+         * Read file data as dataURL using the FileReader API.
+         * @param {type} aFile The loaded file.
+         * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/FileReader#readAsDataURL()}
+         * @returns {undefined}
          */
-        me.readFileData = function (file) {
+        me.readFileData = function (aFile) {
             if (typeof FileReader === 'undefined') {
                 // revert loading status…
-                AttachmentViewer.hideAttachment();
-                AttachmentViewer.hideAttachmentPreview();
-                Alert.showError(I18n._('Your browser does not support uploading encrypted files. Please use a newer browser.'));
+                me.hideAttachment();
+                me.hideAttachmentPreview();
+                Alert.showError('Your browser does not support uploading encrypted files. Please use a newer browser.');
                 return;
             }
 
             var fileReader = new FileReader();
-            if (file === undefined) {
-                file = me.$fileInput[0].files[0];
-                $('#dragAndDropFileName').text('');
+            if (aFile === undefined) {
+                aFile = $fileInput[0].files[0];
+                $($dragAndDropFileName).text('');
             } else {
-                $('#dragAndDropFileName').text(file.name);
+                $($dragAndDropFileName).text(aFile.name);
```

Comment:
`$dragAndDropFileName` should either be already be a jQuery object so it should already be set (in an `init()` method) – I prefer this – or you use a method without prepending the var with `$`, which is actually a string: `'#dragAndDropFileName'`.

So $ is a determinator for a jQuery object, so you must not wrap it in `$(…)` in another time.